### PR TITLE
Implement Explorer UI/UX Mock Up 

### PR DESCRIPTION
**Pull request by mikestonecodes**
##### Description
Explorer UI/UX

##### Checklist
- [x] Linter status: 100% pass
- [x] Changes don't break existing behavior
- [x] Tests coverage hasn't decreased

##### Affected core subsystem(s)
- UI/UX
- Tests

##### Testing
No tests written yet

##### Refers/Fixes
Fixes: #227

##### TODO
 - [x] All contracts drop down
 - [x] Redoing sorting arrows
 - [x] New Pagination UI

### CODE REVIEW

File: src/components/ContractsList.js [hunk 1]
```diff
@@ -304,7 +187,33 @@ class ContractsList extends Component {
         dataIndex: 'EXPIRATION',
         width: 200,
         render: (text, row, index) => {
-          return moment.unix(text).format('Do MMM YYYY, HH:MM:ss');
+          var now = moment();
```
Comment (travisdmathis) [inline src/components/ContractsList.js]: can you break this out into a function in utils so we can reuse it?

**Comment by eswarasai** [comment]
@mikestonecodes -- Thanks for the WIP PR. Let us know once the PR is ready for the review. Also please do not forget to sign the [Contributor License Agreement](https://cla-assistant.io/MARKETProtocol/dApp?pullRequest=246). Thanks!

**Comment by mikestonecodes** [comment]
review needed, See comment in #227 :)

**Comment by eswarasai** [comment]
@mikestonecodes -- Can you please resolve the merge conflicts by rebasing with `upstream:develop`? Please follow the [`CONTRIBUTING`](https://github.com/MARKETProtocol/dApp/blob/develop/.github/CONTRIBUTING.md) guidelines here in case you need help with rebase. Also please do remove the file `.env.local` from your commit. Thanks!

**Comment by mikestonecodes** [comment]
@eswarasai I followed your instructions, thanks :) I'll try to continue to keep up with the changes being made on develop and keep rebasing.

**Comment by mikestonecodes** [comment]
Hey @eswarasai , I completed everything including All contracts dropdown.  Redoing sorting arrows is solid, but Pagination UI is hacky but works. Not sure if all contracts dropdown functionality is what you guys had in mind so let me know :)

File: src/components/ContractsList.js [hunk 2]
```diff
@@ -27,7 +28,14 @@ class ContractsList extends Component {
     sort: { columnKey: 'CONTRACT_NAME', order: 'descend' },
     contracts: this.props.contracts,
     page: 1,
-    pageSize: 25
+    pageSize: 25,
+    selectedContractFilter: 'All Contracts',
+    allContractsFilters: {
+      'All Contracts': contract => contract,
+      'high balance': contract => contract.collateralPoolBalance > 0.5,
```
Comment (travisdmathis) [inline src/components/ContractsList.js]: lets normalize these filter options, either all lower case or capitalize each word.

**Pull request merged**